com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Project.java

```diff
@@ -921,14 +921,53 @@ private void downloadSymbols(IProgress progress) throws IOException, CompileExce
         }
     }
 
+    static void addToPath(String variable, String path) {
+        String newPath = null;
+
+        // Check if jna.library.path is set externally.
```

Comment:
This comment seems very specific while the function name is generic. Same with the mention of texc_shared below